### PR TITLE
nano acknowledgement

### LINKED ISSUE #207: Please provide a link to the original nano repo in README
Many issues are talking about the upstream (orginial?) nano repo. But there are no links anywhere. 
Please add a link to the "upstream" nano repo.

### PR DESCRIPTION
**Pull request by lopho**
link to original nano repo and project fixes #207

### CODE REVIEW

File: Readme.rst [hunk 1]
```diff
@@ -77,3 +77,7 @@ or search package::
     $ aurman -Ss nano-syntax-highlight
 
 Then you need to mannually add ``.nanorc``-s to user's ``~/.nanorc`` or system ``/etc/nanorc``. See **$2**
+
+Acknowledgement
+~~~~~~~~~~
```
Comment (scopatz) [inline Readme.rst]: This line needs to go the full length underneath "Acknowledgement"

**Comment by scopatz** [comment]
Thanks for putting this in @lopho! Let me know when it is updated.

File: Readme.rst (hunk 1 above)
Comment (lopho) [reply Readme.rst]: Like all the other lines in this README? lol.
Ok then. But I'm not your janitor.

**Comment by lopho** [comment]
@scopatz done

**Comment by scopatz** [comment]
Thanks

**Pull request merged**